src/app.rb

```diff
@@ -0,0 +1,64 @@
+require_relative 'storage'
+require_relative 'actions/utiles_helper'
+
+class App
+  def initialize
+    @store = Store.new
+    @B_utiles = BookUtiles.new(@store)
+    @M_utiles = MusicUtiles.new(@store)
+    @Movies_utiles = MovieUtiles.new(@store)
+    @G_utiles = GameUtiles.new(@store)
+  end
+
+  def sub_processor(option)
+    case option
+    when 1
+      @store.books.each do |book|
+        puts "id: #{book.id}, Publisher: #{book.publisher}, Publish Date: #{book.publish_date}, Cover State: #{book.cover_state}, label: #{book.label}"
+      end
+    when 2
+      @store.musics.each do |music|
+        puts "Title: #{music.title}, Publish Date: #{music.publish_date}, on Spotify: #{music.on_spotify}, genre: #{music.genre}"
+      end
+    when 3
+      @store.movies.each do |movie|
+        puts "Silent: #{movie.silent}, Publish Date: #{movie.publish_date}, source: #{movie.source}"
+      end
```

Comment:
Same thing with the source in movies.

![image](https://user-images.githubusercontent.com/38088818/202693688-8d0ef882-113e-4a53-8e83-3b18293b65fb.png)